fabric/fabric-core/src/main/java/io/fabric8/service/FabricServiceImpl.java

```diff
@@ -1335,6 +1336,9 @@ public String getValue(String toSubstitute) {
                         return substituteBundleProperty(toSubstitute, bundleContext);
                     }
                 }));
+                }catch(Exception exception){
```

Comment:
Why don't you just catch `EncryptionOperationNotPossibleException` instead of catcing all?

Reply:
tadayosi, you are right I will try to have a better indentation. Secondly I thought that there might be more such scenario where a different exception might be raised.

I will test with catching EncryptionOperationNotPossibleException and than further update.